src/holodeck/spaces.py

```diff
@@ -84,6 +84,12 @@ def __init__(self, shape, low, high, buffer_shape=None):
     def sample(self):
         return np.random.randint(self._low, self._high, self._shape, dtype=np.int32)
 
+    def get_min(self):
+        return self._low
+
+    def get_max(self):
+        return self._high
+
```

Comment:
Can we call these `get_low` and `get_high`?

Reply:
Done!

Reply:
Done